Sources/MockoloFramework/Models/MethodModel.swift

```diff
@@ -16,85 +16,145 @@
         let nameString = comps.removeFirst()
         self.filePath = filepath
-
+        
         self.data = data
         self.name = nameString
         self.type = Type(ast.typeName) // == .unknownVal ? "" : ast.typeName
-        self.staticKind = ast.isStaticMethod ? .static : ""
+        
+        self.isStatic = ast.isStaticMethod
         self.processed = processed
         self.isInitializer = ast.isInitializer
         self.offset = ast.range.offset
         self.length = ast.range.length
-
+        
         let paramDecls = ast.substructures.filter(path: \.isVarParameter)
         assert(paramDecls.count == comps.count)
         
         let zippedParams = zip(paramDecls, comps)
         self.params = zippedParams.map { (argAst: Structure, argLabel: String) -> ParamModel in
             ParamModel(argAst, label: argLabel, offset: argAst.offset, length: argAst.length, data: data, isInitializer: ast.isInitializer)
         }
-
-        let paramLabels = self.params.map {$0.label != "_" ? $0.label : ""}
-        let paramNames = self.params.map(path: \.name)
-        let paramTypes = self.params.map(path: \.type)
+        
         self.genericTypeParams = ast.substructures
             .filter(path: \.isGenericTypeParam)
             .map { (arg: Structure) -> ParamModel in
                 ParamModel(arg, label: arg.name, offset: arg.offset, length: arg.length, data: data, isGeneric: true)
         }
-        let genericNameTypes = self.genericTypeParams.map { $0.name.capitlizeFirstLetter + $0.type.displayName }
-        var args = zip(paramLabels, paramNames).compactMap { (argLabel: String, argName: String) -> String? in
-            let val = argLabel.isEmpty ? argName : argLabel
-            if val.count < 2 || !nameString.lowercased().hasSuffix(val.lowercased()) {
-                return val.capitlizeFirstLetter
-            }
-            return nil
-        }
-        args.append(contentsOf: genericNameTypes)
-        args.append(contentsOf: paramTypes.map(path: \.displayName))
-        var dtype = self.type.displayName
-        let capped = min(dtype.count, 32)
-        dtype.removeLast(dtype.count-capped)
-        args.append(dtype)
-        
-        // Used to make the underlying function handler var name unique by providing args
-        // that can be appended to the name
-        self.signatureComponents = args.filter{ arg in !arg.isEmpty }
 
         // Sourcekit structure api doesn't provide info on throws/rethrows, so manually parse it here
         let suffixOffset = ast.nameOffset + ast.nameLength + 1
         let suffixLen = ast.offset + ast.length - suffixOffset
         if suffixLen > 0, suffixOffset > ast.bodyOffset - 1 {
-        let suffixPart = data.toString(offset: suffixOffset, length: suffixLen).trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
+            let suffixPart = data.toString(offset: suffixOffset, length: suffixLen).trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
```

Comment:
nit: could we run `swiftformat` on all the files once so we can avoid these indentation fixes ?